.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/tiiuae/PX4-msgs.git
 [submodule "ros2_ws/src/px4_ros_com"]
 	path = ros2_ws/src/px4_ros_com
-	url = https://github.com/tiiuae/px4_ros_com.git
+	url = https://github.com/fly4future/px4_ros_com.git
```

Comment:
We should keep tiiuae submodule, but we shall update tiiuae/px4_ros_com repo with needed changes for f4f nav stack.